docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
```

Comment:
Can we use version 3?

Reply:
uh i can try, would it make a difference?

Reply:
i see no regressions so i don't see why not